spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraph.scala

```diff
@@ -481,6 +481,11 @@ class DnnGraph(
     )
     checkDuplicate()
   }
+
+  override def release(): Unit = {
+    super.release()
+    reorderManager.release()
```

Comment:
the `super.release()` will not release the whole memory.

Reply:
dnn graph is a container, and "super.release()" will release all modules.